framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -51,6 +51,13 @@ namespace bpl {
 #define BPL_OPER_MODE_WDS_REPEATER 3
 #define BPL_OPER_MODE_L2NAT_CLIENT 4
 
+/* Platform Management Mode */
+#define BPL_MGMT_MODE_MULTIAP_CONTROLLER_AGENT 0 /* EasyMesh controller and agent */
+#define BPL_MGMT_MODE_MULTIAP_CONTROLLER 1       /* EasyMesh controller */
+#define BPL_MGMT_MODE_MULTIAP_AGENT 2            /* EasyMesh agent */
+#define BPL_MGMT_MODE_NOT_MULTIAP                                                                  \
+    3 /* Non EasyMesh /*
```

Comment:
Why is this split over two lines?

Reply:
because I accidentally used `/*` instead of `*/` to close the comment.. fixed.